src/synchrotron.jl

```diff
@@ -57,18 +57,18 @@ function S_syn(ϵ, mps)
     # doppler factor D(Γ,θ)
     # Γ is the Bulk Lorentz Factor
     # Θ is the angle between the direction of the blob's motion and the direction to the observer
+    # B = 1 / β in the Dermer e tal. (1997) paper
     β = sqrt(1.0 - 1.0/mps.Γ^2)
     μ_obs = cos(mps.θ)
-    D = 1 / (mps.Γ*(1-(β * μ_obs)))
+    D = 1.0 / (mps.Γ*(1 - μ_obs/β))
```

Comment:
This is the definition of \beta but the Dermer et al. (1997) paper used B which is 1/beta. So I've changed to code to be consistent with their equations (i.e., \mu_obs / \beta = \mu_obs * B).